tests/components/august/test_init.py

```diff
@@ -33,6 +39,40 @@
 )
 
 
+@pytest.fixture(name="config_entry")
+def config_entry_fixture():
+    """Create a mock HEOS config entry."""
+    return MockConfigEntry(
+        domain=DOMAIN,
+        data={
+            CONF_LOGIN_METHOD: "email",
+            CONF_TIMEOUT: 1,
+            CONF_USERNAME: "august",
+            CONF_ACCESS_TOKEN_CACHE_FILE: "mock.file",
+            CONF_PASSWORD: "hidden",
+        },
+        title="August august",
+    )
+
+
+async def test_august_is_offline(hass, config_entry):
+    """Throw ConfigEntryNotReady when august is down."""
+    # Assert setup returns false
+    hass.data[DOMAIN] = {}
+
+    lastex = None
+    with patch(
+        "august.authenticator_async.AuthenticatorAsync.async_authenticate",
+        side_effect=asyncio.TimeoutError,
+    ):
+        try:
+            await august.async_setup_entry(hass, config_entry)
+        except ConfigEntryNotReady as ex:
```

Comment:
This will be caught by the config entries core so we need to assert the config entry state instead. It should be `ENTRY_STATE_SETUP_RETRY`.

Reply:
Thank you for the constant.  This made it possible to find other uses that got me in the right direction.